### DISCUSSION TITLE
Can't change note type to Super any more

### DISCUSSION THREAD

**Discussion by berkano** [standardnotes/forum#3019] [General]
**Describe the bug**
It's no longer possible to change a note type to Super.

**To Reproduce**
1. Create plain note
2. Try to change note type to Super
3. the "Convert to Super note" dialog appears
4. click Convert
5. Result: note does not change type

**Desktop:**
 - OS: Windows 10
 - Browser: Chrome latest, and also desktop app (latest - installed just to test this)
 - Version 3.151.5

**Additional context**
- Happens on web demo, browser version and desktop version
- It was working fine yesterday so, (if it's not a problem with my computer), it could be due to a recent release.

**Comment by effieeee** [comment]
Hi @berkano, I’m afraid I wasn’t able to reproduce this on the latest versions, so we may need to check if there are any account related issues causing this. Can I ask you to contact us at help@standardnotes.com using your Standard Notes account email?

**Comment by berkano** [comment]
Hi @effieeee , thanks for checking this - it's working again today so I am guessing something weird happened on my PC. Thanks!
